### PR TITLE
Teams-fx Migration Toolkit Samples

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/graph-teams-tag/nodejs/teamsapp.local.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile:
+      titleId: M365_TITLE_ID
+      appId: M365_APP_ID
+
+deploy:
+  # Run npm command
+  - uses: cli/runNpmCommand
```
Comment (formulahendry) [inline samples/graph-teams-tag/nodejs/teamsapp.local.yml:93]: need `npm install` for client as well?

File: samples/meetings-context-app/nodejs/server/teamsapp.local.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  # Teams Developer Portal.
+  # Will use the app id in manifest file to determine which Teams app to update.
+  - uses: teamsApp/update
+    with:
+      # Relative path to this file. This is the path for built zip file.
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+
+deploy:
+  # Run npm command
+  - uses: cli/runNpmCommand
```
Comment (formulahendry) [inline samples/meetings-context-app/nodejs/server/teamsapp.local.yml:76]: same here. Need `npm install` for client?

File: samples/meetings-details-tab/nodejs/teamsapp.local.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  # Teams Developer Portal.
+  # Will use the app id in manifest file to determine which Teams app to update.
+  - uses: teamsApp/update
+    with:
+      # Relative path to this file. This is the path for built zip file.
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+
+deploy:
+  # Run npm command
+  - uses: cli/runNpmCommand
```
Comment (formulahendry) [inline samples/meetings-details-tab/nodejs/teamsapp.local.yml:76]: same here. Need `npm install` for client?

File: samples/graph-teams-tag/nodejs/teamsapp.local.yml (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/graph-teams-tag/nodejs/teamsapp.local.yml:93]: fixed

File: samples/meetings-context-app/nodejs/server/teamsapp.local.yml (hunk 2 above)
Comment (Mohammed-MSFT) [reply samples/meetings-context-app/nodejs/server/teamsapp.local.yml:76]: fixed

File: samples/meetings-details-tab/nodejs/teamsapp.local.yml (hunk 3 above)
Comment (Mohammed-MSFT) [reply samples/meetings-details-tab/nodejs/teamsapp.local.yml:76]: fixed

File: samples/graph-pinned-messages/nodejs/teamsapp.local.yml [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile:
+      titleId: M365_TITLE_ID
+      appId: M365_APP_ID
+
+deploy:
+  # Run npm command
+  - uses: cli/runNpmCommand
```
Comment (formulahendry) [inline samples/graph-pinned-messages/nodejs/teamsapp.local.yml:93]: npm install for clientapp

Comment (Mohammed-MSFT) [reply samples/graph-pinned-messages/nodejs/teamsapp.local.yml:99]: fixed

**Review by formulahendry — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
@Mohammed-MSFT - Please run build pipeline.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**